zip-0315.rst

```diff
@@ -4,6 +4,451 @@
   Title: Best Practices for Wallet Handling of Multiple Pools
```

Comment:
We should change this to reflect the broader scope of this document, and also to allow this to become the living spec for wallets to refer to e.g. for fees:
```suggestion
  Title: Best Practices for Wallets
```